app/controllers/api/pods_controller.rb

```diff
@@ -113,8 +113,17 @@ def verify_pushes_allowed!
                       'your version of CocoaPods to push this specification.'
             json_error(422, message)
           end
+
+          if Version.new(version[1]) == Version.new("1.3.0") 
```

Comment:
I think it's something like:
```
spec.recursive_subspecs.any?(&:test_specification?)
```

I don't mind that much either, the biggest bug is if a test spec with 1.3.0 is published *and* is consumed by anything < 1.3.0 it will be treated as a subspec. This means if Google for example publishes a new popular pod then suddenly all apps will be linking test dependencies.